### ISSUE TITLE
Countdown for scheduled job display (cosmetics)

### ISSUE THREAD

**Issue by kwmiebach** [yohamta/dagu#385]
I have a schedule set like this:

```
schedule: "*/5 0-6 * * *"
steps:
  - name: do-something
    command: /bin/bash -c "do something ..."
```

The display of the remaining time till the next run looks inverted, or is this intentional?

In fact 4 hours 21 minutes is  the correct value

![image](https://user-images.githubusercontent.com/6737928/220726523-0e04bcba-33b9-4041-ad16-3a9e4709ec24.png)

**Comment by yohamta** [comment]
Hi, @kwmiebach
This is intentional. I believe it makes the display easier to read, as short times align vertically from left to right (except when the next run starts in less than 1 minute).

**Comment by bbqi** [comment]
"in 4h21m" is much better.       

/admin/src/components/molecules/DAGTable.tsx#L60
```
const durFormatSec = 'd[d]h[h]m[m]s[s]';
const durFormatMin = 'd[d]h[h]m[m';
```

**Comment by yohamta** [comment]
Do you know of any OSS examples that display time in the "in 4h21m" format? I am open to changing the format if it is commonly used, but I have searched for an example in the past and couldn't find one.

**Comment by bbqi** [comment]
zabbix ( open source monitor soultion.)

![image](https://user-images.githubusercontent.com/60316749/224487886-28f4a968-fdb7-4cde-a259-6feaa329888d.png)

**Comment by bbqi** [comment]
search keyword on github: countdown
 
[[hilios](https://github.com/hilios)/[jQuery.countdown](https://github.com/hilios/jQuery.countdown)]
2.3K stars

```
<div id="getting-started"></div>
<script type="text/javascript">
  $('#getting-started').countdown('2015/01/01', function(event) {
    $(this).html(event.strftime('%w weeks %d days %H:%M:%S'));
  });
</script>
```

**Comment by yohamta** [comment]
Thank you for providing the examples. We can change it in the next version `1.10.0`. I think there was a reason why I chose the format, but I can't remember exactly, so maybe it isn't a big deal.

**Comment by yohamta** [comment]
This issue is fixed in `v1.10.1`.
